variants/esp32s3/diy/lilka/platformio.ini

```
@@ -0,0 +1,23 @@
+[env:lilka]
+extends = esp32s3_base
+board = esp32-s3-devkitc-1
+board_build.partitions = default_16MB.csv
+board_check = true
```

```suggestion
board_level = extra
```
board_level must be set to `extra` and board_check must be removed -- for DIY hardware.